hypervisor/arch/x86/gdt.c

```diff
@@ -11,14 +11,14 @@ static void set_tss_desc(union tss_64_descriptor *desc,
 {
 	uint32_t u1, u2, u3;
 
-	u1 = ((uint64_t)tss << 16) & 0xFFFFFFFF;
-	u2 = (uint64_t)tss & 0xFF000000;
-	u3 = ((uint64_t)tss & 0x00FF0000) >> 16;
+	u1 = ((uint64_t)tss << 16) & 0xFFFFFFFFULL;
```

Comment:
I am lost on the usage of UL/ULL. I will stop here to get it clarified

Reply:
Per C99, L (long) is at least 32-bit while LL (long long) is at least 64-bit. The actual bits of each integer type is implementation-defined. Thus I use ULL for 64-bit constants whenever possible.